src/lib.rs

```diff
@@ -332,7 +331,8 @@ pub trait Probe {
 /// * Can produce and merge transparent snapshots
 #[repr(C)]
 pub struct ModalityProbe<'a> {
-    history: &'a mut DynamicHistory<'a>,
+    /// Publicly accessible for direct access by debug-collector
+    pub history: &'a mut DynamicHistory<'a>,
```

Comment:
As noted elsewhere, it seems like we ought to prefer not making the probe internals public (and publicly mutable) if the main goal for doing so is the determination of field offsets for the debug collector (and its opt-in risk profile).